services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -155,42 +155,42 @@ export const ReviewSubmit = ({
             <SubmissionTypeSummarySection
                 submission={draftSubmission}
                 submissionName={submissionName}
-                navigateTo="type"
+                navigateTo="../type"
                 statePrograms={statePrograms}
             />
 
             <ContractDetailsSummarySection
                 submission={draftSubmission}
-                navigateTo="contract-details"
+                navigateTo={'../contract-details'}
```

Comment:
why is this in braces instead of double quotes like above?

Reply:
My mistake.